src/orderbook/index.ts

```diff
@@ -98,17 +121,45 @@ export class StreamedOrderbook {
     )
 
     await orderbook.applyPastEvents()
+
+    if (options.endBlock === undefined) {
+      await orderbook.applyNewEvents()
+      orderbook.scheduleUpdate()
+    }
+
     return orderbook
   }
 
   /**
-   * Dumps the current account state to console.
+   * Stops the orderbook so that it no longer performs any updates.
+   *
+   * @throws
+   * Throws any error that may have occurred when updating and applying new
```

Comment:
I find it a bit weird that this method can potentially throw an error that has happened long in the past and when we are about to end polling anyways. Is there a reason for it? 
Should we maybe just return it or have a bit more information about when and how often this happened?

Reply:
The idea here is that we a polling fiber but no way to check whether or not it error-ed. Eventually, we can:
- Emit "error" events when an error is encountered.
- Also error every time the orderbook is constructed/retrieved from the current state.